pkg/render/fluentd.go

```diff
@@ -335,8 +335,6 @@ func (c *fluentdComponent) container() corev1.Container {
 func (c *fluentdComponent) envvars() []corev1.EnvVar {
 	envs := []corev1.EnvVar{
 		{Name: "FLUENT_UID", Value: "0"},
-		{Name: "ELASTIC_FLOWS_INDEX_SHARDS", Value: "5"},
```

Comment:
Do these variables get set somewhere else that I haven't seen?

Reply:
@matthewdupre Yup. Here: https://github.com/tigera/operator/blob/master/pkg/render/fluentd.go#L445-L446